### PR TITLE
fix library left pane bar - correct num followers & fix on site & couple...

### PR DESCRIPTION
**Pull request by ahsu1230**
... of library SVGs used
@yipingliao @andrewconner @2is10

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/tags/tags.js [hunk 1]
```diff
@@ -74,7 +74,9 @@ angular.module('kifi')
         });
 
         function setTagListHeight() {
-          scrollableTagList.height(w.height() - (scrollableTagList.offset().top - w[0].pageYOffset));
+          if (scrollableTagList.offset() === {}) {
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/tags/tags.js:77]: If `scrollableTagList.offset()` is an empty object, how will it have a `top` property?

File: kifi-backend/modules/shoebox/angular/src/layout/nav/navLibs.styl [hunk 2]
```diff
@@ -34,3 +34,10 @@
   
 .kf-lib-input-box
   padding: 14px 0 10px 0
+
+.library-card
+  width: 27px
+  height: 35px
+  display: inline-block
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/nav/navLibs.styl:41]: List positioning and display properties first.

File: kifi-backend/modules/shoebox/angular/src/tags/tags.js (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/tags/tags.js:77]: ah crap mistype - but the errors don't show anymore

File: kifi-backend/modules/shoebox/angular/src/layout/nav/navLibs.styl (hunk 2 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/layout/nav/navLibs.styl:41]: sure,
I guess display takes precedence over width&height even though they are all part of the BOX property https://smacss.com/book/formatting#grouping

File: kifi-backend/modules/shoebox/angular/src/tags/tags.js (hunk 1 above)
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/tags/tags.js:77]: I wonder why it's working...

Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/tags/tags.js:77]: that will always evaluate to `false`

File: kifi-backend/modules/shoebox/app/com/keepit/model/LibraryMembershipRepo.scala [hunk 3]
```diff
@@ -24,6 +24,7 @@ trait LibraryMembershipRepo extends Repo[LibraryMembership] with RepoWithDelete[
     excludeState: Option[State[LibraryMembership]] = Some(LibraryMembershipStates.INACTIVE))(implicit session: RSession): Seq[LibraryMembership]
   def countWithLibraryIdAndAccess(libraryId: Id[Library], accessSet: Set[LibraryAccess],
     excludeState: Option[State[LibraryMembership]] = Some(LibraryMembershipStates.INACTIVE))(implicit session: RSession): Int
+  def countMemberships(libraryId: Id[Library], excludeState: Option[State[LibraryMembership]] = Some(LibraryMembershipStates.INACTIVE))(implicit session: RSession): Int
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/model/LibraryMembershipRepo.scala:27]: `Memberships` is redundant, implied by the repo type. Consider just `count` (argument makes it clear that it’s for a particular library) or `countWithLibraryId` (like the method above).

**Pull request merged**